src/routes/resources/kendo-japanese-pronunciations-and-definitions/+page.svelte

```diff
@@ -0,0 +1,759 @@
+	optional, it is indicated by parentheses. Many times you will see the character of は (“ha”)
+	written, but if it is used outside of a word it is pronounced “wa” like the character わ. In
+	addition, if the character へ is used in a word it is pronounced “hey”, while if it is used
+	outside of a word in a sentence it is pronounced “eh” identical to え.
+</p>
+<p>
+	Many written words in common usage drop the unemphasized and sometimes silent “u” う. Two examples
+	are “bougu” (bogu) and “kyuu” (kyu). Sometimes it is easier for nonJapanese speakers to think of
+	these “silent” characters as a sign to hold the vowel or consonant a little longer.
+</p>
+<p>
+	For ease of pronunciation and computerized input systems, the phonetic romanization (Hepburn
+	system) is used in this glossary. The complete roman spelling is first, followed by the common
+	spelling in parentheses if different from the phonetic version. The kanji follow the romanized
+	version of the word, and the hiragana phonetic characters are last.
+</p>
+<p>A pronunciation guide follows the glossary if you feel like you need more information.</p>
+
+<h2>A brief note about dialects:</h2>
+<p>
+	There are about one hundred dialects of Japanese, including those that were constructed
+	deliberately. They range from slight differences in pronunciation to differences so pronounced
+	that the dialect is unintelligible for other Japanese speakers from other parts of the country.
+	Please keep this in mind as the terminology from school to school and individual instructors may
+	be different from what is listed here.
+</p>
+
+<h2>A few words about politeness:</h2>
+<p>
+	Japanese has many levels of formality. The ritualized language used within Kendo practice is more
+	formal and polite than everyday language. The general rule is that the longer the phrase, the more
+	polite it is.
+</p>
+<p>
+	As an example, the following phrases all mean “Thank you”, the English translation roughly
+	approximates the level of formality implied by their usage:
+</p>
+<dl>
+	<dt>Doumo (Domo): どうも</dt>
+	<dd>thanks (informal)</dd>
+
+	<dt>Arigato: 有難う【ありがとう】</dt>
+	<dd>thanks (informal)</dd>
+
+	<dt>Doumo arigato: どうも有難う 【どうもありがとう】</dt>
+	<dd>thank you (formal)</dd>
+
+	<dt>Arigato gozaimasu: 有難うご座います【ありがとうございます】</dt>
+	<dd>thank you (formal)</dd>
+
+	<dt>Doumo arigato gozaimasu: どうも有難うご座います【どうもありがとうございます】</dt>
+	<dd>thank you (very much) (formal+)</dd>
+</dl>
+<p>Notice that the phrase used during kendo practice is the “formal+” version.</p>
+
+<h2>Definitions – now the good stuff!</h2>
+
+<h3>People:</h3>
+<dl>
+	<dt>Sensei: 先生 【せんせい】</dt>
+	<dd>teachers, instructors – in kendo usually reserved for someone at 3rd Dan and up</dd>
+
+	<dt>Senpai: 先輩 【せんぱい】</dt>
+	<dd>senior</dd>
+
+	<dt>Kouhai (Kohai): 後輩 【こうはい】</dt>
+	<dd>junior</dd>
+
+	<dt>Kenshi: 剣士 【けんし】</dt>
+	<dd>a practitioner of kendo, fencer</dd>
+
+	<dt>Kendoka: 剣道家【けんどうか】</dt>
+	<dd>a practioner of kendo, fencer</dd>
```

Comment:
These 2 have the same definition but spelled differently

Reply:
They can both be used to refer to a practitioner of kendo, but the literal translations and how often the words are used are different. I just confirmed that is how its written in thhe original document. Since this is a content change, I don't think it's in scope for this PR. I have added it to the checklist for the content issue #190

Reply:
Sounds good do you at least want to fix the spelling error in this PR?

Reply:
Oh yeah, might as well. I forgot about that one, or maybe I missed it haha. Fixed!